solarforecastarbiter/io/reference_observations/default_forecasts.py

```diff
@@ -176,3 +176,45 @@ def is_in_nwp_domain(site):
     TEMPLATE_NWP_FORECASTS +
     TEMPLATE_DETERMINISTIC_PERSISTENCE_FORECASTS
 )
+
+TEMPLATE_NETLOAD_PERSISTENCE_FORECASTS = [
+    Forecast(
+        name='Week Ahead Persistence',
+        issue_time_of_day=dt.time(0),
+        lead_time_to_start=pd.Timedelta('7d'),
+        interval_length=pd.Timedelta('1h'),
+        run_length=pd.Timedelta('1d'),
+        interval_label='ending',
+        interval_value_type='interval_mean',
+        variable='net_load',
+        site=_DUMMY_SITE,
+        extra_parameters=json.dumps(
+            {'is_reference_persistence_forecast': True})
+        ),
+    Forecast(
+        name='Day Ahead Persistence',
+        issue_time_of_day=dt.time(0),
+        lead_time_to_start=pd.Timedelta('1d'),
+        interval_length=pd.Timedelta('1h'),
+        run_length=pd.Timedelta('1d'),
+        interval_label='ending',
+        interval_value_type='interval_mean',
+        variable='net_load',
+        site=_DUMMY_SITE,
+        extra_parameters=json.dumps(
+            {'is_reference_persistence_forecast': True})
+        ),
+    Forecast(
+        name='Hour Ahead Persistence',
+        issue_time_of_day=dt.time(0),
+        lead_time_to_start=pd.Timedelta('1h'),
+        interval_length=pd.Timedelta('1h'),
+        run_length=pd.Timedelta('1h'),
+        interval_label='beginning',
```

Comment:
did you intend for this interval_label to be inconsistent with the others? I'm not opposed - just checking.

Reply:
Actually, I'm just going to make a judgement call and keep everything as `interval_label='ending'` since I think that best matches "real-world" net load forecasting applications.